src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WindowsStreamSecurityUpgradeProvider.cs

```diff
@@ -224,7 +222,9 @@ protected override Stream OnAcceptUpgrade(Stream stream, out SecurityMessageProp
 
                 remoteSecurity = CreateClientSecurity(negotiateStream, _parent.ExtractGroupsForWindowsAccounts);
                 return negotiateStream;
-#endif // FEATURE_NETNATIVE
+#else
+                throw ExceptionHelper.PlatformNotSupported("NegotiateStream is not supported on UWP yet"); 
+#endif // SUPPORTS_WINDOWSIDENTITY
```

Comment:
I'll stop mentioning that we might want to remove UWP from all these PNSE exception messages.